.github/workflows/ci.yml

```
@@ -23,3 +23,5 @@ jobs:
         VALUESET_KEY: ${{ secrets.VALUESET_KEY }}
       run: unzip -P $VALUESET_KEY test/fixtures/EXM130-8.0.000/EXM130-8.0.000-bundle.zip
     - run: rm EXM130-8.0.000-bundle.json
+    - run:  sudo chmod +x ./_tests_/setup.sh
```

```suggestion
    - run: sudo chmod +x ./_tests_/setup.sh
    - run: ./_tests_/setup.sh
```
The `setup.sh` needs to also be run.